CHANGELOG.md

```diff
@@ -7,10 +7,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.10.0-dev
 
+### Added
+
+- Option `colors.transparent_background_colors` to allow applying opacity to all background colors
+
 ### Changed
 
 - `ExpandSelection` is now a configurable mouse binding action
 
+### Removed
+
+- Config option `background_opacity`, you should use `window.opacity` instead
```

Comment:
I don't think we added things to the `Removed` section in the past until it was actually removed? I think this should just be a `Changed` entry.